src/tools/bin/plotNumericalHeating

```diff
@@ -119,7 +119,7 @@ for sim in directories:
     mydir = sim+simDir
     # get relevant files with energy
     files = [f for f in os.listdir(mydir)
-             if os.path.isfile(os.path.join(mydir, f)) and re.search('^.*_energy_all.dat', f)]
+             if os.path.isfile(os.path.join(mydir, f)) and (re.search('^.*_energy_all.dat', f) or re.search('^.fields_energy.dat', f))]
     # check if file list is empty
     if len(files) == 0:
         sys.exit("There were no energy files in \"{}\".".format(mydir))
```

Comment:
I wonder if we should also at least print a warning of the field energy is missing.

Reply:
But in the case where I will only plot the lose of energy of an moving particle species (maybe one particle) it is not needed.
A suggestion is to rename this to something like `plotGlobalHeating` and enforce at least one particle species and the field energy.